lxqt-config-appearance/lxqtthemeconfig.ui

```diff
@@ -51,6 +51,13 @@
      </column>
     </widget>
    </item>
+   <item>
+    <widget class="QCheckBox" name="wallpaperOverride">
+     <property name="text">
+      <string>Override independently set wallper</string>
```

Comment:
But it's not only about using it... it's about overriding... 
Even when unchecked the wallpaper is changed (if the pre-existing wallpaper is set by the theme).